herbert_core/classes/instrument_classes/@IX_map/section.m

```diff
@@ -1,49 +1,43 @@
-function map_out=section(map,index)
-% Create a new map object by defined by an index array
+function obj_out = section(obj, wkno_keep_in)
+% Create a new IX_map by selecting particular workspaces
```

Comment:
appreciating that the terms spectra and workspace may be familiar to users of map, they are not defined here and it may be useful to point the user to documentation

Reply:
I've added a description to the classdef file IX_map.m that explains what a map, spectrum and workspace are. This is display with >> doc IX_map. I think that is sufficient in the present context; I would assume that if a user is looking at a method of a class then they would turn to the class description to get more details, rather than explicitly suggest they do so here.

Your comment does raise a more general one about function documentation: we don't have consistency across classes, and most certainly insufficient documentation across the Horace documentation about concepts and conventions. One for post-construction...